piptools/repositories/pypi.py

```diff
@@ -31,7 +31,7 @@
 from pip._internal.req import InstallRequirement, RequirementSet
 from pip._internal.req.req_tracker import get_requirement_tracker
 from pip._internal.utils.hashes import FAVORITE_HASH
-from pip._internal.utils.logging import indent_log, setup_logging
+from pip._internal.utils.logging import RichPipStreamHandler, indent_log, setup_logging
```

Comment:
oversaw that this won't work with old pip versions - needs to be imported depending on version